sdk/Lusid.Sdk.Tests/Utilities/InstrumentExamples.cs

```diff
@@ -48,7 +48,7 @@ internal static LusidInstrument CreateExampleFxOption()
                 instrumentType: LusidInstrument.InstrumentTypeEnum.FxOption
             );
         
-        internal static LusidInstrument CreateExampleEquityOption(bool isCashSettled)
+        internal static EquityOption CreateExampleEquityOption(bool isCashSettled = false)
```

Comment:
Do we need to add this to GetExampleInstrument() up above?

Reply:
I would obsolete the function, or comment this somewhere. Otherwise I would assume this was a mistake.

Reply:
It can't be obsoleted right now as it's still used in tests. Eh on second thought I changed it to LusidInstrument, no harm done.

Reply:
Will adding the obsolete attribute to the method cause the tests to fail? If we are moving away from generic tests then we should have some way of notifying devs to stop using it.

Reply:
Won't cause anything to fail, the method is still manually called everywhere.